src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/InProcess/SuppressedIssuesFileSynchronizerTests.cs

```diff
@@ -146,14 +145,12 @@ public async Task UpdateFileStorage_FileStorageIsUpdatedOnBackgroundThread()
                 suppressedIssuesFileStorage: suppressedIssuesFileStorage.Object,
                 activeSolutionBoundTracker: activeSolutionBoundTracker.Object);
 
-            testSubject.UpdateFileStorage();
+            await testSubject.UpdateFileStorageAsync();
 
-            threadHandling.Verify(x => x.Run(It.IsAny<Func<Task<Task<bool>>>>()), Times.Once);
-            suppressedIssuesFileStorage.Invocations.Count.Should().Be(0);
-            activeSolutionBoundTracker.Invocations.Count.Should().Be(0);
-
-            await fileUpdateTask();
+            threadHandling.Verify(x => x.SwitchToBackgroundThread(), Times.Once);
 
+            suppressedIssuesFileStorage.Invocations.Count.Should().Be(1);
+            activeSolutionBoundTracker.Invocations.Count.Should().Be(1);
```

Comment:
I'm not sure why the test was not expecting the methods to be called before. However, they are called now.